azurelinuxagent/common/conf.py

```diff
@@ -146,7 +146,8 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
     "Debug.CgroupDisableOnQuotaCheckFailure": True,
     "Debug.EnableAgentMemoryUsageCheck": False,
     "Debug.EnableFastTrack": True,
-    "Debug.EnableGAVersioning": True
+    "Debug.EnableGAVersioning": True,
+    "Debug.EnablePolicy": True
```

Comment:
Let's make this name specific to extensions. Maybe "Debug.EnableExtensionsPolicy"?

Reply:
Updated!